<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2007</width>
    <height>1603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Camera1_Preview">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>561</width>
      <height>316</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>1151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>1151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>20</width>
      <height>671</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>100</y>
      <width>20</width>
      <height>671</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>120</y>
      <width>741</width>
      <height>272</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="captureFormatLabel">
       <property name="text">
        <string>PixelFormat:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="captureFormatOptions"/>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="saveLocation">
       <property name="text">
        <string>Save Location</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="saveFilename">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="exposureLabel">
       <property name="text">
        <string>Exposure</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="exposureLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="frameRateLabel">
       <property name="text">
        <string>FrameRate</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="frameRateLineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Background1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>940</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>930</y>
      <width>20</width>
      <height>361</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>930</y>
      <width>20</width>
      <height>361</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>930</y>
      <width>20</width>
      <height>361</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Tracking1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>1120</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Camera2_Preview">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>100</y>
      <width>561</width>
      <height>316</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>920</y>
      <width>1121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>100</y>
      <width>20</width>
      <height>671</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Camera3_Preview">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>561</width>
      <height>316</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>760</y>
      <width>1151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Camera4_Preview">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>430</y>
      <width>561</width>
      <height>316</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Background2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>940</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Background3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>930</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Background4">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>930</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_10">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>930</y>
      <width>20</width>
      <height>361</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_11">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>930</y>
      <width>20</width>
      <height>361</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_12">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>1100</y>
      <width>1121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_14">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>1280</y>
      <width>1121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Tracking2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>1120</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Tracking3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>1120</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Tracking4">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>1120</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>30</y>
      <width>561</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="SetCameraButton">
       <property name="text">
        <string>SetCamera</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveOptions">
       <property name="text">
        <string>Save Options</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadOptions">
       <property name="text">
        <string>Load Options</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="connectedCameras">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>500</y>
      <width>554</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Connected</string>
    </property>
   </widget>
   <widget class="QToolButton" name="frameGrabButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>820</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Grab Background</string>
    </property>
    <property name="text">
     <string>Tracking</string>
    </property>
   </widget>
   <widget class="QToolButton" name="previewButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Preview</string>
    </property>
   </widget>
   <widget class="QToolButton" name="recordButton">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>790</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Record</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1530</x>
      <y>790</y>
      <width>311</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Compression Type</string>
    </property>
    <widget class="QRadioButton" name="compression_h265">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>H.265 Codec</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="compression_h264">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>241</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>H.264 Codec</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="compression_raw">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>No </string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2007</width>
     <height>39</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Camera</string>
    </property>
   </widget>
   <addaction name="menuCamera"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
